vanessaspear/passport-api
# Created database fixtures to see the database for development

Closes #4 

## Description

Created fixtures for all the MVP tables in the ERD.  Added data to all fixtures.  Created and ran migrations.  Connected to database and confirmed data loaded properly. 

## Steps to Test 

- Run the below commands to setup the local database
```
git fetch origin vs-database-fixtures
git checkout vs-database-fixtures
python3 manage.py makemigrations passportapi
python3 manage.py migrate
python3 manage.py loaddata users tokens itineraries itinerary_categories itinerary_purpose  travel_reasons  trip_notes  trip_reasons trips
```
- Connect to the database and verify tables were created and data was successfully loaded by Django ORM
